server.go

```diff
@@ -45,9 +74,9 @@ func handle(w http.ResponseWriter, r *http.Request) {
 }
 
 func main() {
-	appPrefix := "metadeck"
```

Comment:
I host other stuff on this domain. Is there a better way to route requests on certain paths to this app and other paths to other apps?